src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -1044,76 +1027,147 @@ private AdsDataProvider() {
-            223909212,
-            300,
-            250
+            new Page("adtest", "SyntheticTests/Oasis/FloatingMedrecOnLongPage/300x600"),
+            "",
+            ImmutableMap.<String, Object>builder()
+                .put("slotName", AdsContent.FLOATING_MEDREC)
+                .put("slotSize", new Dimension(300, 600))
+                .put("lineItemId", 223968492)
+                .put("src", "gpt")
+                .build()
         },
         {
-            "adtest",
-            "SyntheticTests/Oasis/FloatingMedrecOnLongPage/160x600",
-            237935652,
-            160,
-            600
+            new Page("adtest", "SyntheticTests/Oasis/FloatingMedrecOnLongPage/OneSkyscraper"),
+            "",
+            ImmutableMap.<String, Object>builder()
+                .put("slotName", AdsContent.FLOATING_MEDREC)
+                .put("slotSize", new Dimension(300, 600))
+                .put("lineItemId", 223968492)
+                .put("src", "gpt")
+                .build()
         },
-    };
-  }
-
-  @DataProvider
-  public static Object[][] adsIncontentLeaderboard() {
-    return new Object[][]{
         {
-            "adtest",
-            "SyntheticTests/INCONTENT_LEADERBOARD/728x90",
-            239961972,
-            728,
-            90
+            new Page("adtest", "SyntheticTests/Oasis/FloatingMedrecOnLongPage/NoSkyScrapers"),
+            "",
+            ImmutableMap.<String, Object>builder()
+                .put("slotName", AdsContent.FLOATING_MEDREC)
+                .put("slotSize", new Dimension(300, 250))
+                .put("lineItemId", 223909212)
+                .put("src", "gpt")
+                .build()
         },
         {
-            "adtest",
-            "SyntheticTests/INCONTENT_LEADERBOARD/300x250",
-            239961132,
-            300,
-            250
+            new Page("adtest", "SyntheticTests/Oasis/FloatingMedrecOnLongPage/NoSkyScrapersWithJumboMedrec"),
+            "",
+            ImmutableMap.<String, Object>builder()
+                .put("slotName", AdsContent.FLOATING_MEDREC)
+                .put("slotSize", new Dimension(300, 250))
+                .put("lineItemId", 223909212)
+                .put("src", "gpt")
+                .build()
         },
         {
-            "adtest",
-            "SyntheticTests/INCONTENT_LEADERBOARD/468x60",
-            241472412,
-            468,
-            60
+            new Page("adtest", "SyntheticTests/Oasis/FloatingMedrecOnLongPage/160x600"),
+            "",
+            ImmutableMap.<String, Object>builder()
+                .put("slotName", AdsContent.FLOATING_MEDREC)
+                .put("slotSize", new Dimension(160, 600))
+                .put("lineItemId", 237935652)
+                .put("src", "gpt")
+                .build()
+        },
+        {
+            new Page("project43", "SyntheticTests/Slots/Size/120x600"),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "SyntheticTests/Slots/Size/120x600" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)